cowamm/profitability/competitor_kpis/curve/curve_kpis_4232873.sql

```diff
@@ -0,0 +1,31 @@
+-- Computes volume, tvl and APR for Curve pools
+-- APR is measured as the fees earned per $ invested, over the last 24 hours, projected over 1 year
+-- Input: blockchain
+select
+    r.contract_address,
+    sum(amount_usd) as volume,
+    365 * sum(amount_usd * fee / tvl) as apr,
+    avg(fee) as fee,
+    avg(tvl) as tvl
```

Comment:
Are we do we want the avg fee and tvl here (this is a number that may have never existed as a true TVL)? I personally think the latest values for fee and tvl would make more sense.

Reply:
The avg(tvl) was to remain consistent with the other pools where the displayed tvl is averaged over the past period. 
This comes form query_4059213. 
In my opinion, it does make sense, as it removes the noise in the tvl generated by small variations.